teamscale_client/client.py

```diff
@@ -334,6 +344,79 @@ def add_baseline(self, baseline):
         service_url += baseline.name
         return self.put(service_url, parameters={}, data=to_json(baseline))
 
+    def get_projects(self):
+        """Retrieves a list of projects from the server.
+
+        Returns:
+            List[:class:`data.ProjectInfo`]): The list of projects.
+
+        Raises:
+            ServiceError: If anything goes wrong
+        """
+        service_url = self.get_global_service_url("projects")
+        parameters = {
+            "detail": True
+        }
+        headers = {'Accept': 'application/json'}
+        response = requests.get(service_url, params=parameters, auth=self.auth_header, verify=self.sslverify,
+                                headers=headers, timeout=self.timeout)
+        if response.status_code != 200:
+            raise ServiceError("ERROR: GET {url}: {r.status_code}:{r.text}".format(url=service_url, r=response))
+        return [
+            ProjectInfo(project_id=x['id'], name=x['name'], description=x['description'],
+                        creation_timestamp=x['creationTimestamp'], alias=x.get('alias'),
+                        deleting=x['deleting'], reanalyzing=x['reanalyzing']) for x in response.json()]
+
+    def add_project(self, project_configuration, only_config_update=False):
+        """Adds a project to Teamscale. Re-adding a project with an existing id will update the original project
+        if `only_config_update` is set to `True`.
```

Comment:
What does only update the config mean? Does this change the running project? If no, what do I have to do to actually cause a change? Maybe expand the documentation a bit?